resources/cards/models/card.yml

```diff
@@ -0,0 +1,108 @@
+Card:
+  title: Card
+  required:
+  - account_id
+  - url
+  - raw_url
+  - front_original_url
+  - thumbnails
+  - mode
+  type: object
+  properties:
+    id:
+      title: Id
+      pattern: card_[a-zA-Z0-9]+$
+      type: string
+      example: card_75b803ba
+    account_id:
+      title: Account Id
+      type: string
+    description:
+      title: Description
+      maxLength: 255
+      type: string
+    url:
+      title: Url
+      maxLength: 2083
+      minLength: 1
+      type: string
+      format: uri
+    size:
+      title: Size
+      type: string
+      default: 2.125x3.375
+    auto_reorder:
+      title: Auto Reorder
+      type: boolean
+      default: false
+    reorder_quantity:
+      title: Reorder Quantity
+      type: integer
+    raw_url:
+      title: Raw Url
+      maxLength: 2083
+      minLength: 1
+      type: string
+      format: uri
+    front_original_url:
+      title: Front Original Url
+      maxLength: 2083
+      minLength: 1
+      type: string
+      format: uri
+    back_original_url:
+      title: Back Original Url
+      maxLength: 2083
+      minLength: 1
+      type: string
+      format: uri
+    thumbnails:
+      title: Thumbnails
+      type: array
+      items:
+        $ref: '#/components/schemas/Thumbnail'
+    available_quantity:
+      title: Available Quantity
+      type: integer
+      default: 0
+    pending_quantity:
+      title: Pending Quantity
+      type: integer
+      default: 0
+    countries:
+      title: Countries
+      type: array
+      items:
+        type: string
+    deleted:
+      title: Deleted
+      type: boolean
+      default: false
+    status:
+      title: Status
+      enum:
+      - processed
+      - rendered
+      type: string
+    mode:
+      title: Mode
+      enum:
+      - test
+      - live
```

Comment:
This line made me think at `all` might also be an acceptable value 🤷 
https://github.com/lob/lob-api/blob/main/src/validators/cards/list.js#L16